server/services/validation/errorDetailAdapter.js

```diff
@@ -30,13 +30,7 @@ const detailAdapterBuilder = firstFieldNameMap => {
  */
 const extractFirstFieldNameMap = R.pipe(
   R.propOr({}, 'keys'),
-  R.map(
-    R.pipe(
-      R.prop('metas'),
-      R.mergeAll,
-      R.prop('firstFieldName')
-    )
-  ),
+  R.map(R.pipe(R.prop('metas'), R.mergeAll, R.prop('firstFieldName'))),
```

Comment:
seems like prettier is a bit broken here.. couldn't work out how to fix to use existing behaviour